Blazor_Foundation_6_Test_Shared/Documentation/Media/Badge.razor

```diff
@@ -1,7 +1,64 @@
 ﻿@page "/badge";
+@using OpenCodeDev.Blazor.Foundation.Doc.Core.Components.Containers.Section;
 
-<h3>Badge</h3>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Badge</h2>
+        <p>The badge is a basic component that displays a number. It's useful for calling out a number of unread items.</p>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Basics</h2>
+        <p>
+            Add the .badge class to an element to create a badge. In the below example, we're using <Code Content="PHNwYW4+"></Code>, but any tag will work fine.
```

Comment:
Add the .badge class should be in <Code> component

Reply:
```suggestion
            Add the <Code>.badge</Code> class to an element to create a badge. In the below example, we're using <Code Content="PHNwYW4+"></Code>, but any tag will work fine.
```

Reply:
Remove . before class